### ISSUE TITLE
NVDA doesn't read drop-down selected value in Firefox. Working fine in Chrome and Edge.

### ISSUE THREAD

**Issue by NeerajVishwakarma** [nvaccess/nvda#12329]
### Steps to reproduce:
    1. Open Firefox
    2. Open NVDA with/out Browse mode
    2. Go to Drop-Down on the page
    3. Select Value
    
    Chrome, Edge work fine. 
    
### Actual behavior:
    Does not announce selected value. Instead it reads Label + combobox + collapsed
### Expected behavior:
    NVDA should read the selected value Label + combobox + selected value + collapsed
#### NVDA installed/portable/running from source:
    Yes installed
#### NVDA version:
    2019.2.1
#### Windows version:
    Win 10
#### Does the issue still occur after restarting your computer?
    Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
    No
#### If add-ons are disabled, is your problem still occurring?
    Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
    No

**Comment by XLTechie** [comment]
Can you provide an example page please?

**Comment by Adriani90** [comment]
This is fixed since Firefox 109. Closing as works for me.
